common/lib/dependabot/pull_request_creator.rb

```diff
@@ -189,7 +189,7 @@ def bitbucket_creator
         pr_description: message.pr_message,
         pr_name: message.pr_name,
         author_details: author_details,
-        labeler: labeler,
+        labeler: nil,
```

Comment:
For posterity, it seems that [BitBucket doesn't support labels](https://jira.atlassian.com/browse/BCLOUD-11976) by default. There are [plugins that enable that functionality](https://labels.reconquest.io/docs/pull-request-labels-add-on-overview/). This seems like an appropriate default 👍